File: CursoPOOUber/Python/creditCard.py
from payment import Payment

class CreditCard(Payment):

  nameCard = str
  cardNumber = int
  dateCard = int
  cvv = int

  def __init__(self, id, nameCard, cardNumber, dateCard, cvv):
      super().__init__(id)
      self.nameCard = nameCard
      self.cardNumber = cardNumber
      self.dateCard = dateCard
      self.dateCard = dateCard
      self.cvv = cvv

File: CursoPOOUber/Java/CreditCard.java
class CreditCard extends Payment {
  String nameCard;
  Integer cardNumber;
  String dateCard;
  Integer cvv;

  public CreditCard(Integer id, String nameCard, Integer cardNumber, String dateCard, Integer cvv) {
    super(id);
    this.nameCard = nameCard;
    this.cardNumber = cardNumber;
    this.dateCard = dateCard;
    this.cvv = cvv; 
  }
}
